src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -278,58 +278,46 @@ def get_suggested_assignees(
     return []
 
 
-def get_suspect_commit_block(project: Project, event: GroupEvent):
-    """Build up the suspect commit info for the given event"""
-    author = None
-    commit_id = None
-    commit_link = None
-    pr_link = None
-    pr_date = None
-    committers = None
-    repo_base = None
+def get_suspect_commit_text(
+    project: Project, event: GroupEvent, commits: Sequence[Mapping[str, Any]] | None = None
+) -> SlackBlock:
+    """Build up the suspect commit text for the given event"""
 
-    try:
-        committers = get_serialized_event_file_committers(project, event)
-    except (Release.DoesNotExist, Commit.DoesNotExist):
-        logger.info("Skipping suspect committers because release does not exist.")
-    except Exception:
-        logger.exception("Could not get suspect committers. Continuing execution.")
-    if not committers:
+    # commits is passed from context when the rule initially fires
+    # we may not have that data if the message is being built after an action is taken, for example
+    if not commits:
+        commits = get_commits(project, event)
+    if not commits:
         return None
 
-    if committers:
-        committer = committers[0]
-        author = committer.get("author", {})
-        commits = committer.get("commits")
-        if commits:
-            commit_id = commits[0].get("id")
-            pull_request = commits[0].get("pullRequest")
-            if pull_request:
-                pr_title = pull_request.get("title")
-                pr_link = pull_request.get("externalUrl")
-                repo_base = pull_request.get("repository", {}).get("url")
-                pr_date = pull_request.get("dateCreated")
-                pr_id = pull_request.get("id")
-                if pr_date:
-                    pr_date = time_since(pr_date)
-
-    if author and commit_id:
-        author_display = (
-            author.get("name") if author.get("name") is not None else author.get("email")
-        )
+    commit = commits[0]  # get the most recent commit
+    suspect_commit_text = "Suspect Commit: "
+    pull_request = commit.get("pull_request")
+    author = commit.get("author")
+    commit_id = commit.get("id")
+    if not author and not commit_id:
+        return None
```

Comment:
should this be `or`?

Reply:
I need both to continue, did I do it wrong 🤔